### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Welcome_AzureDataStudio-Insiders_Zoom:The data at 400% is not clearly visible.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#10617]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_AzureDataStudioStayHealthy;#A11ySev2;#A11yTCS;#A11yWCAG2.1;#Benchmark;#DesktopApp;#WCAG1.4.10;#SQL Azure Data Studio;#FTP;#Zoom;#Win32;#Acc123;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.19.0-insider
Commit: e8dc0d1
Date: 2020-05-27T17:24:57.733Z
VS Code: 1.46.0
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:WCAG1.4.10

**Pre-requisite:** set the display resolution to 1280*1024 and advanced scaling to 400% and sign out and launch application

 **Repro Steps:**
1 .Launch Azure Data Studio Insiders application.
2 . If Welcome screen is not shown on application launch, then navigate to Help --> Welcome.

**Actual:**
The four links and the information it has is not visible, the preview dialog information is not fully visibel.

**Issue observed in the following screens:**
Resources
Videos Pop Up
Extend your data Studio
Notebook-Markdown Toolbar
Dashboard-Databases
Dashboard-Home - Toolbar
Notebook-Toolbar 
Python Dependencies Wizard

**Expected:**
All data should be visible clearly when scrolling through the screen or navigating using tab key.

**User Impact:**
Keyboard users will not be able to view all data present when the screen is magnified and access the information present on the click of controls.

**Attachment link for Reference:**
![Screenshot (152)](https://user-images.githubusercontent.com/24580570/83236484-36450e00-a1b1-11ea-9901-6fe4503b32c4.png)
![Screenshot (153)](https://user-images.githubusercontent.com/24580570/83236497-393ffe80-a1b1-11ea-9419-0e3ff9d453e0.png)
![Screenshot (154)](https://user-images.githubusercontent.com/24580570/83236501-3cd38580-a1b1-11ea-8d44-a13a4fde3b9c.png)
![Screenshot (155)](https://user-images.githubusercontent.com/24580570/83236517-42c96680-a1b1-11ea-9f60-71604b4d4ca1.png)

**Comment by alanrenmsft** [comment]
closing the issue, Azure Data Studio requested exception for WCAG 2.1: https://ceacc.visualstudio.com/ceaccex/_workitems/edit/42844/

**Comment by fsteffi** [comment]
@alanrenmsft Reflow Exception link
https://ceacc.visualstudio.com/ceaccex/_workitems/edit/42844

**Comment by Roopesh-T** [comment]
Issue is not repro, 
![image](https://user-images.githubusercontent.com/86601127/218740106-a7db099e-0833-4591-8cb9-b7bbb57347fc.png)
![image](https://user-images.githubusercontent.com/86601127/218740117-5e3c3639-b4c1-4c14-9f5e-66f6017a12e6.png)
![image](https://user-images.githubusercontent.com/86601127/218740129-39acacb0-02f0-45f5-87e6-f8067f985013.png)
As per the new guidelines on the above issue is not repro so closing the bug.
![image](https://user-images.githubusercontent.com/86601127/218740360-f25985c6-3671-4463-b690-ff87a2d69325.png)


**GitHub Tags**:#Closed;
